### PR TITLE
just some small service docs advancements

### PR DESCRIPTION
**Pull request by freben**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
...
     "build:api-reports": "yarn build:api-reports:only --tsc",
-    "build:api-reports:only": "backstage-repo-tools api-reports --allow-warnings 'packages/core-components,plugins/+(catalog|catalog-import|git-release-manager|jenkins|kubernetes)' -o ae-wrong-input-file-type --validate-release-tags",
+    "build:api-reports:only": "NODE_OPTIONS=--max-old-space-size=8192 backstage-repo-tools api-reports --allow-warnings 'packages/core-components,plugins/+(catalog|catalog-import|git-release-manager|jenkins|kubernetes)' -o ae-wrong-input-file-type --validate-release-tags",
```
Comment (freben) [inline package.json:20]: I was starting to hit this ceiling every now and then - let me know if you prefer not to add this

**Review by camilaibs — APPROVED** [review]
Left minor suggestions

File: packages/backend-plugin-api/src/services/definitions/AuthService.ts [hunk 2]
```diff
@@ -18,29 +18,52 @@ import { PermissionAttributes } from '@backstage/plugin-permission-common';
 import { JsonObject } from '@backstage/types';
 
 /**
+ * Represents a user principal (for example when a user Backstage token issued
```
[the remark this answers is not in the document]
Comment (camilaibs) [reply packages/backend-plugin-api/src/services/definitions/AuthService.ts:21]: Same for the [httpAuth](https://backstage.io/docs/backend-system/core-services/http-auth) and [userInfo](https://backstage.io/docs/backend-system/core-services/user-info).

[the remark this answers is not in the document]
Comment (freben) [reply packages/backend-plugin-api/src/services/definitions/AuthService.ts:21]: Yep indeed, the PR was getting so large already 😅

**Comment by freben** [comment]
awesome suggestions, let's get all of those in

**Pull request merged**